.github/dependabot.yml

```diff
@@ -3,33 +3,7 @@ updates:
 
   - package-ecosystem: "github-actions"
     directory: "/"
-    schedule:
-      interval: "weekly"
-    assignees:
-      - "@cartoncloud/code-owners-platform"
-    reviewers:
-      - "@cartoncloud/code-owners-platform"
-
-  - package-ecosystem: "github-actions"
-    directory: "/deployment-status"
-    schedule:
-      interval: "weekly"
-    assignees:
-      - "@cartoncloud/code-owners-platform"
-    reviewers:
-      - "@cartoncloud/code-owners-platform"
-
-  - package-ecosystem: "github-actions"
-    directory: "/jira-deployment-status"
-    schedule:
-      interval: "weekly"
-    assignees:
-      - "@cartoncloud/code-owners-platform"
-    reviewers:
-      - "@cartoncloud/code-owners-platform"
-
-  - package-ecosystem: "github-actions"
-    directory: "/slack-deployment-status"
```

Comment:
Might still be needed, unsure

Reply:
I'm not the author of the code, @aarontwf is.  My best guess is just allowing overrides on a directory level, so no objection to refactoring if they intend to remain consistent

Reply:
I think it was only picking thing up in its direct `.github` folder or `actions.yml`